src/guiFormSpecMenu.cpp

```diff
@@ -1706,6 +1706,76 @@ bool GUIFormSpecMenu::parseSizeDirect(parserData* data, std::string element)
 	return true;
 }
 
+bool GUIFormSpecMenu::parsePositionDirect(parserData * data, std::string element)
+{
+	if (element == "")
+		return false;
+
+	std::vector<std::string> parts = split(element, '[');
+
+	if (parts.size() != 2)
+		return false;
+
+	std::string type = trim(parts[0]);
+	std::string description = trim(parts[1]);
+
+	if (type != "position")
+		return false;
+
+	parsePosition(data, description);
+
+	return true;
+}
+
+void GUIFormSpecMenu::parsePosition(parserData * data, std::string element)
```

Comment:
please look at the code style rules, pointers should not add extra space after asterisk